zebra-chain/src/primitives/zcash_primitives.rs

```diff
@@ -124,3 +124,23 @@ pub(crate) fn sighash(
         .as_ref(),
     )
 }
+
+/// Compute the authorizing data commitment of this transaction as specified
+/// in [ZIP-244].
+///
+/// # Panics
+///
+/// If passed a pre-v5 transaction.
+///
+/// [ZIP-244]: https://zips.z.cash/zip-0244.
+pub(crate) fn auth_digest(trans: &Transaction) -> AuthDigest {
+    let alt_tx: zcash_primitives::transaction::Transaction = trans
+        .try_into()
+        .expect("zcash_primitives and Zebra transaction formats must be compatible");
+    let digest_bytes: [u8; 32] = alt_tx
```

Comment:
A personal style nit, you may choose to ignore:

```suggestion

    let digest_bytes: [u8; 32] = alt_tx
```